backend/coins/btc/addresses/address.go

```diff
@@ -47,22 +47,28 @@ type AccountAddress struct {
 }
 
 // NewAccountAddress creates a new account address.
+// The chainIndex is 0 for receive and 1 for change.
```

Comment:
It's not clear what "chainIndex" is here; it's not one of the argument (though I assume you refer to the one accessed via `derivation.SimpleChainIndex()`? ) maybe we can make it clearer

Reply:
Ah it's a leftover comment from before I introduced the Derivation struct. Removed.